submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,382 @@
+
+{% include image.html
+
+src= 'src'
+ alt='Processing uranium-235 into uranium fuel cells'
+ caption='Processing uranium-235 into uranium fuel cells'
+
+%}
+
+Hence, we see that we can get up to hundreds of times more energy from the same piece of uranium-235 if it is turned into uranium fuel cells, instead of being turned into nuclear fuel. This would suggest that if one already has a uranium-235 supply, they should just switch to nuclear power and electric machines, instead of using burner devices. However, it is also worth noting that nuclear fuel is unlocked alongside Kovarex Enrichment, which allows for the easy production of uranium-235. This means that using uranium efficiently stops being an issue around the time you unlock nuclear fuel.
+
+## Module effects on energy gains
+
+So far we have done our calculations without using any modules for the sake of keeping things simple and understanding the default conditions. Now let's look at them again with modules and see if our results change significantly. Modules primarily affect the power usage of the fuel processing machines. Let's first review the equation:
+
+`Total processing energy (E) = Machine power usage (P) * time of one crafting cycle (t_c) * numnber of crafting cycles (n) / crafting speed (C)`
+
+`E = P * t_c * n / C`
+
+Here is what the modules do:
+
+- **Efficiency modules:** They reduce the power usage of the fuel processing machines ("P"). Their overall effects on energy gains are small but noteworthy for most recipes. I would definitely recommend using efficiency 1 modules in all of the fuel processing machines if you don't use any other modules, because they are quite cheap to make and they cut down power usage by 60% or even 80% (the efficiency limit), depending on how many module slots the machines have.
+
+- **Speed modules:** One effect of speed modules is increasing power usage ("P") just like efficiency modules, but in the opposite direction, hence reducing our energy gains. The other effect of speed modules is increasing the crafting speed of machines ("C"), which as seen from the equation, reduces the power usage because the added speed makes the machine work for a shorter time for each crafting cycle. For every speed module, the increase in P is greater than the increase in C, which means that the overall effect of any speed module is an increase in E, which results in decreases in energy gains overall. However, speed modules can be used alongside productivity modules to mitigate the slowdown they cause, and this might provide energy gains overall.
```

Comment:
```suggestion
- **Speed modules:** One effect of speed modules is increasing power usage ("P") just like efficiency modules, but in the opposite direction, hence reducing our energy gains. The other effect of speed modules is increasing the crafting speed of machines ("C"), which as seen from the equation, reduces the power usage, because the added speed makes the machine work for a shorter time for each crafting cycle. For every speed module, the increase in P is greater than the increase in C, which means that the overall effect of any speed module is an increase in E, which results in decreases in energy gains overall. However, speed modules can be used alongside productivity modules to mitigate the slowdown they cause, and this might provide energy gains overall.
```